src/components/common/TabWrapper.tsx

```diff
@@ -64,9 +64,13 @@ type TabWrapperProps = {
 
 function TabWrapper(props: TabWrapperProps) {
   const { classes, routes, snapshotAccessRequests } = props;
+  const [selectedTab, setSelectedTab] = React.useState<string>('datasets');
+  const location = useLocation();
 
-  const locationSplit = history.location.pathname.split('/');
-  const selectedTab = `/${locationSplit[1] || 'datasets'}`;
+  useEffect(() => {
+    const locationSplit = history.location.pathname.split('/');
```

Comment:
Could this use `location` instead? Is there a difference between `history.location` vs `location`? If so, that would make it clearer why the `useEffect()` is needed here.
```suggestion
    const locationSplit = location.pathname.split('/');
```